src/Control/Lens/At.hs

```diff
@@ -454,17 +458,25 @@ instance At (Maybe a) where
   {-# INLINE at #-}
 
 instance At (IntMap a) where
+#if MIN_VERSION_containers(0,5,8)
```

Comment:
> Do you have strong feelings about it?

No, certainly not! I only added it since you mentioned `IntMap`, and I was naïvely trying to catch all of the cases where `containers-0.5.8.1` added something that could be used in `lens`. I'll revert this particular change for now.